signal_generator_node:
  ros__parameters:
    type_int: 0
    time_float: 0.0
    offset_float: 0.0
    frequency_float: 1.0
    amplitude_float: 0.5